### PR TITLE
install libxss1 to fix missing libXss.so.1

### PR DESCRIPTION
**Pull request by philippotto**
Also see https://github.com/puppeteer/puppeteer/issues/6192#issuecomment-657522885

### CODE REVIEW

**Comment by daniel-wer** [comment]
In the linked PR, a maintainer mentioned that people should switch to `google-chrome-stable` and the docs were adapted accordingly. We could also do that to avoid such problems in the future. I'd guess that we don't need `libxss1`, then.

**Comment by philippotto** [comment]
Yeah, I read that, too, but the comments didn't really confirm (only suggested) that switching from unstable to stable really fixes this particular issue. But we can certainly try it out. Would be the more stable route in the long-term. I'll open a second PR.

**Review by jstriebel — APPROVED** [review]
Waiting for result from #20, but LGTM :+1:

File: puppeteer/Dockerfile
Comment (philippotto) [inline puppeteer/Dockerfile:10]:
```suggestion
    && apt-get install -y libxss1 google-chrome-stable fonts-ipafont-gothic fonts-wqy-zenhei fonts-thai-tlwg fonts-kacst ttf-freefont \
```

**Comment by philippotto** [comment]
There still seems to be an issue (see https://app.circleci.com/pipelines/github/scalableminds/webknossos/2095/workflows/d7696ec9-b772-47f9-913d-91872a55c513/jobs/13296) with the missing libXss.so which is why I'd like to merge this PR, too. I've left chrome at `stable` for now. If this doesn't work for us (maybe there will be new problems), we can switch to `unstable` again. Agree?

**Comment by jstriebel** [comment]
Sure :+1: :shipit:

**Pull request merged**
